examples/my_circuit.rs

```diff
@@ -0,0 +1,100 @@
+use std::{array, num::NonZeroUsize};
+
+use sirius::{
+    ivc::{
+        step_circuit::{trivial, AssignedCell, ConstraintSystem, Layouter},
+        SynthesisError,
+    },
+    prelude::{
+        bn256::{new_default_pp, C1Affine, C1Scalar, C2Affine, C2Scalar},
+        CommitmentKey, PrimeField, StepCircuit, IVC,
+    },
+};
+
+/// Number of folding steps
+const FOLD_STEP_COUNT: usize = 5;
+
+/// Arity : Input/output size per fold-step for primary step-circuit
+const A1: usize = 5;
+
+/// Arity : Input/output size per fold-step for secondary step-circuit
+/// For tivial case it can be any number
+const A2: usize = 1;
+
+/// Table size for Primary Circuit
+///
+/// Requires at least 17, for service purposes, but if the primary requires more, increase the
+/// constant
+const SECONDARY_CIRCUIT_TABLE_SIZE: usize = 17;
+
+/// Key size for Primary Circuit
+const PRIMARY_COMMITMENT_KEY_SIZE: usize = 21;
+
+/// Table size for Primary Circuit
+///
+/// Requires at least 17, for service purposes, but if the primary requires more, increase the
+/// constant
+const PRIMARY_CIRCUIT_TABLE_SIZE: usize = 17;
+
+/// Key size for Primary Circuit
+const SECONDARY_COMMITMENT_KEY_SIZE: usize = 21;
+
+#[derive(Debug, Clone)]
+struct MyConfig {}
+struct MyStepCircuit {}
+
+impl<const A: usize, F: PrimeField> StepCircuit<A, F> for MyStepCircuit {
+    /// This is a configuration object that stores things like columns.
+    type Config = MyConfig;
+
+    /// Configure the step circuit. This method initializes necessary
+    /// fixed columns and advice columns, but does not create any instance
```

Comment:
this will be changed later. we could add a TODO and refer to public io issue. so we will change the comment. it is easily forgot.